### PR TITLE
It is spelled 'acknowledgement'

### LINKED ISSUE #4440: Acknowledgement or acknowledgment, not both
The English and US spelling forms are used arbitrarily in transport draft.

### PR DESCRIPTION
**Pull request by martinthomson**
Closes #4440.

### CODE REVIEW

**Comment by mikkelfj** [comment]
Do I read this wrong, or did you go for acknowledgment contrary to the topic header?

**Comment by martinthomson** [comment]
I was trained on something approximating British English, which generally uses the longer form.  The title is a statement of (subjective) truth with no actual bearing on the changes.

**Comment by mikkelfj** [comment]
In that case the capitalised (capitalized) version might also want to disagree with the topic header: "Acknowledgement", assuming that US spelling is somewhat consistent.

**Review by ianswett — APPROVED** [review]
So it is

**Review by janaiyengar — APPROVED** [review]
Is it though? :-)

I'm fine with either, and I actually prefer the shorter one, but your issue title suggests the opposite change.

File: draft-ietf-quic-qpack.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ Count in order to identify which dynamic table entries can be referenced without
 potentially blocking a stream.  The decoder tracks the Known Received Count in
 order to be able to send Insert Count Increment instructions.
 
-A Section Acknowledgement instruction ({{header-acknowledgement}}) implies that
+A Section Acknowledgement instruction ({{header-acknowledgment}}) implies that
```
Comment (mikkelfj) [inline draft-ietf-quic-qpack.md:346]:
```suggestion
A Section Acknowledgment instruction ({{header-acknowledgment}}) implies that
```

**Comment by mikkelfj** [comment]
There are other places with uppercase, but that needs a search, e.g. https://github.com/quicwg/base-drafts/blob/1fa5d4958609a9c1eb6ed2d94d16415476428b6e/draft-ietf-quic-http.md#acknowledgements

**Comment by martinthomson** [comment]
Someone should learn to drive sed better.  I've fixed the uppercase variants as well.

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -1041,17 +1041,17 @@ Consider the following sequence of events:
 |:-------|:---------------------------|
 | t=0    | Send packet #1 (app data)  |
 | t=1    | Send packet #2 (app data)  |
-| t=1.2  | Recv acknowledgement of #1 |
+| t=1.2  | Recv acknowledgment of #1 |
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md:1044]:
```suggestion
| t=1.2  | Recv acknowledgment of #1  |
```

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -1041,17 +1041,17 @@ Consider the following sequence of events:
...
 | t=2    | Send packet #3 (app data)  |
 | t=3    | Send packet #4 (app data)  |
 | t=4    | Send packet #5 (app data)  |
 | t=5    | Send packet #6 (app data)  |
 | t=6    | Send packet #7 (app data)  |
 | t=8    | Send packet #8 (PTO 1)     |
 | t=12   | Send packet #9 (PTO 2)     |
-| t=12.2 | Recv acknowledgement of #9 |
+| t=12.2 | Recv acknowledgment of #9 |
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md:1052]:
```suggestion
| t=12.2 | Recv acknowledgment of #9  |
```

**Review by janaiyengar — APPROVED** [review]
couple of formatting ones, but otherwise lgtm.

File: draft-ietf-quic-recovery.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ normative:
 informative:
 
   FACK:
-    title: "Forward Acknowledgement: Refining TCP Congestion Control"
+    title: "Forward Acknowledgment: Refining TCP Congestion Control"
```
Comment (MikeBishop) [inline draft-ietf-quic-recovery.md:67]: Is the title of this paper in fact different from what we had in the text?

**Review by MikeBishop — APPROVED** [review]
Now I know why we used "ack" so much -- to avoid deciding which way to spell this.

**Comment by ianswett** [comment]
> Now I know why we used "ack" so much -- to avoid deciding which way to spell this.

No kidding, and acknowledgement is pretty long.  I wonder if we should just define 'ack' as acknowledgement at the beginning.

**Comment by MikeBishop** [comment]
As "acknowledgment," you mean.  😉  I prefer it with the 'e', but without is technically the American spelling.  (Although what I found said both occur everywhere; the only difference is which is more prevalent.)

File: draft-ietf-quic-recovery.md (hunk 4 above)
Comment (MikeBishop) [reply draft-ietf-quic-recovery.md:67]: However, they use "Acknowledgment" without the 'e' in the abstract.  Yeesh.

**Comment by mikkelfj** [comment]
> one more table

The gift that keeps on giving ...

**Review by afrind — APPROVED** [review]

**Pull request merged**